terracumber_config/tf_files/SUSEManager-4.2-PRV.tf

```diff
@@ -189,8 +189,14 @@ module "cucumber_testsuite" {
     kvm-host = {
       image = "sles15sp3o"
       additional_grains = {
-        hvm_disk_image = "http://minima-mirror.mgr.prv.suse.net/distribution/leap/15.3/appliances/openSUSE-Leap-15.3-JeOS.x86_64-OpenStack-Cloud.qcow2"
-        hvm_disk_image_hash = "http://minima-mirror.mgr.prv.suse.net/distribution/leap/15.3/appliances/openSUSE-Leap-15.3-JeOS.x86_64-OpenStack-Cloud.qcow2.sha256"
+        hvm_disk_image = {
+          leap = "http://minima-mirror.mgr.prv.suse.net/distribution/leap/15.3/appliances/openSUSE-Leap-15.3-JeOS.x86_64-OpenStack-Cloud.qcow2"
```

Comment:
same remark as in the sumaform PR: any reason for remaining on 15.3 ?

Reply:
This is currently the default image. I did not want to update this in my PR but do follow up PR for that.